sw/device/lib/arch/device.h

```diff
@@ -51,9 +51,19 @@ typedef enum device_type {
 extern const device_type_t kDeviceType;
 
 /**
- * The clock frequency of the device, in hertz.
+ * The cpu clock frequency of the device, in hertz.
  */
-extern const uint64_t kClockFreqHz;
+extern const uint64_t kClockFreqCpuHz;
+
+/**
+ * The peripheral clock frequency of the device, in hertz.
```

Comment:
Would you mind elaborating on this comment? What is the "peripheral clock", exactly? Same goes for the others (though they're mildly more obvious I feel...

Reply:
i added some description, can you let me know it's clear enough